vmdb/app/models/host.rb

```diff
@@ -1751,6 +1753,7 @@ def scan_from_queue(taskid=nil)
     $log.info("#{log_header} Scanning #{log_target}...")
 
     task.update_status("Active", "Ok", "Scanning") if task
+    self.authentication_status_by_parent = "Valid" if self.kind_of?(HostOpenstackInfra) && self.verify_credentials
```

Comment:
This is completely apart from my concern about having the reference to `HostOpenstackInfra` inside the base class `Host`.

@Fryguy, this also plays around our discussion today with JHardy about moving openstack infrastructure type things to cloud.